app/javascript/components/organizations-dropdown.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <clickable-dropdown
+    :body-title="dropdownTitle"
+    :no-items-message="noOrganizationsMessage"
+    :items="organizations_with_name"
+    :default-index="defaultOrganizationIndex"
+    @item-clicked="onItemClicked"
+  >
+  </clickable-dropdown>
+</template>
+
+<script>
+import ClickableDropdown from './clickable-dropdown';
+import { PROCESS_NEW_ORGANIZATION } from '../store/action-types';
+import FroggoLocalStorage from '../helpers/local-storage';
+
+export default {
+  props: {
+    organizations: Array,
+    githubLogin: String,
+    teams: Array,
+  },
+  data() {
+    const named_organizations = this.organizations;
+    named_organizations.forEach(function(item) {
+      item.name = item.login;
+    });
+    return {
+      dropdownTitle: this.$t('message.profile.organizationsDropdownTitle'),
+      noOrganizationsMessage: this.$t('message.profile.noOrganizations'),
+      organizations_with_name: named_organizations,
```

Comment:
Identifier 'organizations_with_name' is not in camel case  camelcase<br>Identifier 'named_organizations' is not in camel case      camelcase